src/MathMagiciansApp.jsx

```diff
@@ -1,5 +1,31 @@
+import {
+  NavLink, Route, Routes,
+} from 'react-router-dom';
 import Calculator from './components/Calculator';
+import Home from './components/Home';
+import Quote from './components/Quote';
 
-const MathMagiciansApp = () => <Calculator />;
+const MathMagiciansApp = () => (
```

Comment:
- [ ] Good job so far :+1:  But I noticed that you rename your `App.js`, I think it will be better to keep the `App.js`. `App.js` on the other hand has the root component of the react app because every view and component are handled with hierarchy in React, where <App /> is the top component in the hierarchy. Please rename this file name to `App.js` :smile: